corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -116,6 +116,7 @@ <h4 class="modal-title">{% trans "Create a new Case Type" %}</h4>
   {% initial_page_data 'typeChoices' question_types %}
   {% initial_page_data 'fhirResourceTypes' fhir_resource_types %}
   {% initial_page_data 'read_only_mode' request.is_view_only %}
+  {% url 'geospatial_settings' domain as geospatial_settings_url %}
```

Comment:
No, this is intentional and the `geospatial_settings` url gets used on line 434 in `<a href="{{ geospatial_settings_url }}">`. 

The reason I put the url at the top instead of in the `href` property is because the `blocktrans` tag doesn't allow having other block tags inside of it.